spec/models/schedule_spec.rb

```diff
@@ -11,4 +11,53 @@
     it { is_expected.to validate_presence_of(:applies_from) }
     it { is_expected.to validate_presence_of(:applies_to) }
   end
+
+  describe "instance methods" do
```

Comment:
can we avoid this extra nesting please, as adding `.name` defines a test for an instance method?

Reply:
Usually `#name` for instance and `.name` for class.